rssfeed/models.py

```diff
@@ -1,53 +1,50 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
 
 class Feed(models.Model):
     title = models.CharField(max_length=2000, blank=True, null=True)
-    xml_url = models.CharField(max_length=255, unique=True)
-    link = models.CharField(max_length=2000, blank=True, null=True)
+    url = models.CharField(max_length=2000, unique=True)
     description = models.TextField(blank=True, null=True)
-    published_time = models.DateTimeField(blank=True, null=True)
-    last_polled_time = models.DateTimeField(blank=True, null=True)
+    link = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        db_index=True
+    )
+    published = models.DateTimeField(blank=True, null=True)
+    last_polled = models.DateTimeField(blank=True, null=True)
+    image = models.ImageField(max_length=2000, null=True)
 
     class Meta:
-        verbose_name = "Feed"
-        verbose_name_plural = "Feeds"
+        verbose_name = _("Feed")
+        verbose_name_plural = _("Feeds")
 
     def __unicode__(self):
-        return self.title or self.xml_url
+        return self.title or self.url
 
     def save(self, *args, **kwargs):
-        """Poll new Feed"""
+        # Poll new Feed
         try:
-            Feed.objects.get(xml_url=self.xml_url)
+            Feed.objects.get(url=self.url)
             super(Feed, self).save(*args, **kwargs)
         except Feed.DoesNotExist:
             super(Feed, self).save(*args, **kwargs)
-            from rssfeed.utils import poll_feed
-
-            poll_feed(self)
-
-
-class EntryManager(models.Manager):
-    def number_unread(self):
-        return Entry.objects.filter(is_read=False).count()
+            from rssfeed.tasks import poll_feed
```

Comment:
The import is here because the `poll_feed` function requires a Feed object, AND an Entry object, and since at this point, only the Feed object has been instantiated, which means if the import is global, it will only see the Feed instantiation, and not the Entries, and will error.